naga/xtask/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "xtask"
 version = "0.1.0"
 edition = "2021"
 publish = false
+rust-version = "1.83"
```

Comment:
Couln't this inherit from the workspace?

Reply:
No, the xtasks aren't part of the workspace for reasons I don't currently remember

Reply:
`xtasks` are deliberately kept separate from the main workspace. We _can_ include them, but (warning: unqualified perf. claim) that can add to build time in hot development loops. We haven't encountered any significant problems for not tying this to the workspace, minus the question of the `rust-version` here, AFAIK?